File: data/imported/knowledge/slides/java/merged/snippets/snippet_010.java
@Test
    void testRemoveAuthor() throws IOException {
        try (var pres = new Presentation()) {
            pres.getCommentAuthors().addAuthor("Alice", "A");
            pres.getCommentAuthors().addAuthor("Bob", "B");
            pres.getCommentAuthors().remove(pres.getCommentAuthors().get(0));
            assertThat(pres.getCommentAuthors().size()).isEqualTo(1);

            try (var pres2 = saveAndReopen(pres)) {
                assertThat(pres2.getCommentAuthors().size()).isEqualTo(1);
                assertThat(pres2.getCommentAuthors().get(0).getName()).isEqualTo("Bob");
            }
        }
    }

File: data/imported/knowledge/slides/net/merged/snippets/snippet_010.cs
[Fact]
    public void TestRemoveAuthor()
    {
        // Removing an author persists.
        using var pres = new Presentation();
        pres.CommentAuthors.AddAuthor("Alice", "A");
        pres.CommentAuthors.AddAuthor("Bob", "B");
        pres.CommentAuthors.Remove(pres.CommentAuthors[0]);
        pres.CommentAuthors.Count.Should().Be(1);

        using var pres2 = TestHelpers.SaveAndReopen(pres, _tempDir);
        pres2.CommentAuthors.Count.Should().Be(1);
        pres2.CommentAuthors[0].Name.Should().Be("Bob");
    }